### DISCUSSION TITLE
Reverse the "Animation Speed" slider

### DISCUSSION THREAD

**Discussion by shirok1** [mulaRahul/keyviz#51] [General]
**Describe the solution you'd like**
The slider labeled "Animation Speed" actually controls animation duration, which means left end represents fast and right end does the else, counterintuitive for a "Speed" slider.

**Describe alternatives you've considered**
...or simply relabel the slider with "Animation Duration".

**Additional context**
Even 200ms is considered too slow by me, hopefully more options can be added.

**Comment by mulaRahul** [comment]
Indeed it can confuse users. I've already planned to change the settings UI, either slider will be replaced with an input/_spinbox_ or swap the start and end values of the slider to match the label.

> Even 200ms is considered too slow by me, hopefully more options can be added.

Maybe I'll set 100ms as the lower limit for animation speed.
